File: lib/src/main/java/uk/iatom/rfm69/registers/values/RxTimeout2.java
package uk.iatom.rfm69.registers.values; import uk.iatom.rfm69.registers.IRegisterValue;

public enum RxTimeout2 implements IRegisterValue {
    
    // RegRxTimeout2),
    RSSITHRESH_VALUE(0x00); // Default),;

    private byte val;
    
    RxTimeout2(int val) {
        this.val = (byte) val;
    }

    @Override public byte val() {
        return val;
    }
}

File: lib/src/main/java/uk/iatom/rfm69/registers/values/RxTimeout2.kt
package uk.iatom.rfm69.registers.values

import uk.iatom.rfm69.registers.IRegisterValue

enum class RxTimeout2(`val`: Int): IRegisterValue { // RegRxTimeout2),
RSSITHRESH_VALUE(0x00);


    // Default),;
    private val `val`: Byte

    init {
        this.`val` = `val`.toByte()
    }

    override fun `val`(): Byte {
        return `val`
    }
}
